workflow/pred_temperature_ridge.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<p>I add more information about RGDR, and the reason that we take only one target interval and one lag. Also the link to the tutorial notebook of RGDR, which clearly explains the usecase with "target_intervals" and the idea behind it, is added in this part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>